packages/atlas/src/views/studio/VideoWorkspace/VideoWorkspace.tsx

```diff
@@ -117,10 +117,10 @@ const VideoWorkspaceActionBar: React.FC<VideoWorkspaceActionBarProps> = ({
   onResize,
 }) => {
   const mdMatch = useMediaMatch('md')
-  const [actionBarRef, actionBarBounds] = useMeasure()
+  const { ref: actionBarRef, height: actionBarBoundsHeight = 1 } = useResizeObserver()
```

Comment:
This gets measured as 48px. Seems this library by default measure only the content-box, so padding doesn't get included in the measurement. Check https://github.com/ZeeCoder/use-resize-observer#options This will need to be taken into account in other places as well